pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -185,7 +187,14 @@ func Run() {
 		}
 	}
 
-	domainCtx := domainContext{usbAccess: true}
+	// These settings can be overridden by GlobalConfig
+	domainCtx := domainContext{
```

Comment:
Shouldn't these be specified as defaults in global.go?? Why do we need to hardcode them here?